packages/admin/src/pages/technologies/Form/ReviewersForm.js

```diff
@@ -0,0 +1,172 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@material-ui/core';
+import DeleteIcon from '@material-ui/icons/Delete';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+	SimpleShowLayout,
+	SimpleForm,
+	SelectInput,
+	useNotify,
+	useRedirect,
+	useDataProvider,
+	SaveButton,
+	Toolbar,
+	ReferenceInput,
+	useQuery,
+	ArrayField,
+	Datagrid,
+	TextField,
+	Loading,
+	DateField,
+	ReferenceField,
+} from 'react-admin';
+
+const useStyles = makeStyles((theme) => ({
+	deleteButton: {
+		margin: theme.spacing(1),
+		backgroundColor: 'red',
+	},
+}));
+
+const ReviewersForm = ({ record, resource, basePath }) => {
+	const historic = useQuery({
+		type: 'getList',
+		resource: `technologies/${record.id}/reviewer-history`,
+		payload: {
+			pagination: {
+				page: 1,
+				perPage: 100,
+			},
+			sort: {
+				field: 'id',
+				order: 'asc',
+			},
+		},
+	});
+
+	const [reviewer, setReviewer] = useState();
+	const [loading, setLoading] = useState(true);
+	const classes = useStyles();
+	const CustomToolbar = () => {
+		const notify = useNotify();
+		const redirect = useRedirect();
+		const dataProvider = useDataProvider();
+		const handleSubmit = () => {
+			setLoading(true);
+			dataProvider
+				.update(`${resource}/${record.id}/reviewer`, { id: '', data: { reviewer } })
+				.then(() => {
+					notify('ra.notification.updated', 'info', { smart_count: 1 });
+					redirect(basePath);
+				})
+				.catch(() => {
+					notify('ra.notification.http_error', 'warning');
+					setLoading(false);
+				});
+		};
+		const deleteAction = () => {
+			setLoading(true);
+			dataProvider
+				.update(`${resource}/${record.id}/disassociate-reviewer`, { id: '', data: {} })
+				.then(() => {
+					notify('ra.notification.updated', 'info', { smart_count: 1 });
+					redirect(basePath);
+				})
+				.catch(() => {
+					notify('ra.notification.http_error', 'warning');
```

Comment:
Depois vê se seria mt custoso passar isso aqui pra um enum (pode ser noutra PR mesmo)
```suggestion
					notify(notifications.http_error, 'warning');
```